src/components/Layout/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { StaticQuery, graphql } from 'gatsby';
+import { Helmet } from 'react-helmet';
+import PropTypes from 'prop-types';
+import Header from './Header';
+import Footer from './Footer';
+import favicon from '../../images/favicon.ico';
+import '../../stylesheets/app.scss';
+
+export default (props) => (
+  <StaticQuery
+    query={graphql`
+      query SiteQuery {
+        site {
+          siteMetadata {
+            title
+            description
+          }
+        }
+      }
+    `}
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    render={(data) => <Layout data={data} {...props} />}
+  />
+);
+
+
+const Layout = (props) => {
+  // Define the meta title and description
+  const { data } = props;
+  const { title } = data.site.siteMetadata;
+  const { description } = data.site.siteMetadata;
+  const { children } = props;
```

Comment:
```suggestion
const Layout = ({ data, children }) => {
  // Define the meta title and description
  const { title, description } = data.site.siteMetadata;
```

You can destructure multiple properties at the same time. You can also destructure the arguments directly in the function parameters list.

Reply:
Thanks!